README.md

```diff
@@ -402,6 +403,50 @@ account.signOut().then(function (sessionProperties) {
 })
 ```
 
+### account.destroy
+
+Destroys the account of the currently signed in user.
+
+```js
+account.destroy()
+```
+
+Resolves with `sessionProperties` like [account.signin](#accountsignin),
+but without the session id:
+
+```json
+{
+  "account": {
+    "id": "account123",
+    "username": "pat",
+    "createdAt": "2016-01-01T00:00.000Z",
+    "updatedAt": "2016-01-02T00:00.000Z",
+    "profile": {
+      "fullname": "Dr. Pat Hook"
+    }
+  }
+}
+```
+
+Rejects with:
+
+<table>
+  <tr>
+    <th align="left"><code>Error</code></th>
+    <td><em>A custom error thrown in a <code>before:destroy</code> hook</em></td>
+  </tr>
+</table>
```

Comment:
It can also fail with `ConnectionError`, because an actual request needs to be sent to destroy an account, which is not necessary for a signOut